rx/__init__.py

```diff
@@ -246,47 +275,56 @@ def empty(scheduler: Optional[typing.Scheduler] = None) -> Observable:
         >>> obs = rx.empty()
 
     Args:
-        scheduler: Scheduler to send the termination call on.
+        scheduler: [Optional] Scheduler instance to send the termination call
+            on. By default, this will use an instance of
+            :class:`ImmediateScheduler <rx.scheduler.ImmediateScheduler>`.
 
     Returns:
         An observable sequence with no elements.
     """
+
     from .core.observable.empty import _empty
     return _empty(scheduler)
 
 
-def for_in(values: Iterable[Any], mapper: Mapper) -> Observable:
+def for_in(values: Iterable[Any], mapper: typing.Mapper) -> Observable:
     """Concatenates the observable sequences obtained by running the
-    specified result mapper for each element in source.
+    specified result mapper for each element in the specified values.
 
     .. marble::
         :alt: for_in
 
-        a--1--2-|
-         b--10--20-|
+        a----1----2--|
+         b----10---20--|
         [for_in((a, b), lambda i: i+1)]
-        ---2--3--11--21-|
+        -----2----3----11----21--|
```

Comment:
Not sure here, but I think there should be 6 hyphens instead of 4 between `3` and `11`, i.e. 2 from the end of 'a' observable + 4 from the start of the 'b' observable.

Reply:
```
        a----1----2--|
         b----10---20--|
        [for_in((a, b), lambda i: i+1)]
        -----2----3------11----21--|
```
Like this?

Reply:
Exactly :+1: